File: 백준/Bronze/14681. 사분면 고르기/사분면 고르기.cc
#include <stdio.h>

int Quadrant(int x, int y) {
	if (x > 0 && y > 0) {
		return 1;
	}
	else if (x < 0 && y > 0) {
		return 2;
	}
	else if (x < 0 && y < 0) {
		return 3;
	}
	else if (x > 0 && y < 0) {
		return 4;
	}

	return 0;
}

int main(void) {
	int x = 0, y = 0;

	if (scanf("%d %d", &x, &y) != 2) return -1;

	int result = Quadrant(x, y);

	printf("%d\n", result);

	return 0;
}

File: 백준/Bronze/14681. 사분면 고르기/사분면 고르기.py
import sys
input = sys.stdin.readline

def Quadrant(x, y):
    if x > 0 and y > 0:
        return 1
    elif x < 0 and y > 0:
        return 2
    elif x < 0 and y < 0:
        return 3
    elif x > 0 and y < 0:
        return 4

def solve():
    x = int(input().rstrip())
    y = int(input().rstrip())
    
    result = Quadrant(x, y)
    
    print(result)

if __name__ == "__main__":
    solve()
